Makefile

```diff
@@ -322,17 +353,21 @@ create-clientconfig:
 	done
 
 
-CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
-controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0)
+KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+.PHONY: kustomize
```

Comment:
.PHONY should be in every Makefile target or none I guess. Now the behavior changes per target. I prefer every target, since we're not dealing with files (other than the $(LOCALBIN) target)

Reply:
Yeah this was already inconsistent before it seems.
I can do it here, or on a separate PR if we don't want to mix the changes.

Reply:
Separate PR is fine.

Reply:
#697